src/Cafe/HW/Latte/Core/LatteShaderCache.cpp

```diff
@@ -52,6 +52,11 @@ struct
 	sint32 pipelineFileCount;
 }g_shaderCacheLoaderState;
 
+struct
+{
+	ImTextureID textureId;
+}d_shaderCacheLoaderState;
```

Comment:
Creating a new structure seems unnecessary. You can just add `textureId` to `g_shaderCacheLoaderState`.
I also suggest renaming both the TV and GamePad `textureId` to something more descriptive like `bootTVTexId` and `bootDRCTexId` which avoids the name collision and matches the filenames.